packages/flutter_tools/lib/src/ios/plist_parser.dart

```diff
@@ -28,6 +28,33 @@ class PlistParser {
   static const String kCFBundleShortVersionStringKey = 'CFBundleShortVersionString';
   static const String kCFBundleExecutable = 'CFBundleExecutable';
 
+  /// Returns the content, converted to XML, of the plist file located at
+  /// [plistFilePath].
+  ///
+  /// If [plistFilePath] points to a non-existent file or a file that's not a
+  /// valid property list file, this will return null.
+  ///
+  /// The [plistFilePath] argument must not be null.
+  String? plistXmlContent(String plistFilePath) {
```

Comment:
I don't think this is used outside this library.
```suggestion
  String? _plistXmlContent(String plistFilePath) {
```

Reply:
I actually intend to override this internally to use `plistutil` so that `PlistParser` can be used on Linux.